webpack.config.js

```diff
@@ -8,20 +9,32 @@ module.exports = {
 	devtool: 'sourcemap',
 	stats: 'errors-only',
 	entry: {
-		background: './source/background',
-		options: './source/options'
+		background: './source/background.js',
+		options: './source/options.js'
 	},
 	output: {
 		path: path.join(__dirname, 'distribution'),
 		filename: '[name].js'
 	},
+	module: {
+		rules: [
+			{
+				test: /\.css$/,
+				use: [
+					MiniCssExtractPlugin.loader,
+					'css-loader'
+				]
+			}
+		]
+	},
 	plugins: [
+		new MiniCssExtractPlugin(),
 		new SizePlugin(),
 		new CopyWebpackPlugin([
 			{
 				from: '**/*',
 				context: 'source',
-				ignore: ['*.js']
+				ignore: ['*.js', '*.css']
 			},
```

Comment:
If you prefer not having to use `MiniCssExtractPlugin`, you can just set `CopyWebpackPlugin` to copy the raw CSS file like this:
```suggestion
			},
			{
				from: 'node_modules/webext-base-css/webext-base.css'
			}
```

And then just load it in `options.html`
```html
<link rel="stylesheet" href="chrome://global/skin/in-content/common.css">
<link rel="stylesheet" href="webext-base.css">
<link rel="stylesheet" href="options.css">
```